client-guide/agile-workflows.html.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Agile workflows
+---
+
+The single biggest problem we have seen in development teams is the lack of an established
+philosophy and process for managing the day-to-day work of the developers and ensuring they are
+delivering value to the business without requiring a constant back and forth. Bureaucracy is
+sacrificed in the name of speed, but this leads to teams that only perform at a fraction of their
+potential, often leaving stakeholders unhappy.
```

Comment:
- The sentence is long-ish, breaking it up or making it more concise would help readability.
- Why they only perform at a fraction of their potential?
- Maybe stress that bureaucracy vs. speed is a fine balance, going too far in either direction hurts team performance in the short or long run.